packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -26,7 +26,7 @@ const execAppOptions: ExecSyncOptions = {
 };
 const registry = 'http://localhost:4873';
 
-const packageManager = process.env.TEST_PACKAGE_MANAGER || 'npm';
```

Comment:
I've put yarn as default here for the moment, waiting for the PR that fixes shell for npm